### PR TITLE
Fix uncommitted code error.

### PR DESCRIPTION
**Pull request by nmatyukov**
When I tried to install react_on_rails, I got "ERROR: You have uncommitted code. Please commit or stash your changes before continuing." but all of my code were committed. 

The problem was in the git localization: "нечего фиксировать, рабочая директория пуста" instead of "nothing to commit, working directory clean" (but both are the same).

Here is the fix that checks on empty git status if no uncommitted changes.

[Review on Reviewable](https://reviewable.io/reviews/shakacode/react_on_rails/303)

### CODE REVIEW

File: lib/react_on_rails/git_utils.rb [hunk 1]
```diff
@@ -2,8 +2,8 @@ module ReactOnRails
   module GitUtils
     def self.uncommitted_changes?(message_handler)
       return false if ENV["COVERAGE"]
-      status = `git status`
-      return false if status.include?("nothing to commit, working directory clean")
```
Comment (justin808) [inline lib/react_on_rails/git_utils.rb:6]: @nmatyukov Sweet!
@robwise CC

**Comment by justin808** [comment]
:lgtm: Thanks!

---

Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/shakacode/react_on_rails/303#-:-KBt25zT-4Ef_OBSKpjE:-971291639)*

**Comment by justin808** [comment]
@nmatyukov Can you please rebase this on top of master. Then I'll have the auto-merge button.

**Comment by nmatyukov** [comment]
@justin808 Branch was rebased.

**Comment by justin808** [comment]
Thanks @nmatyukov!

File: lib/react_on_rails/git_utils.rb [hunk 2]
```diff
@@ -2,8 +2,8 @@ module ReactOnRails
...
+      status = `git status --porcelain`
```
Comment (mamantoha) [inline lib/react_on_rails/git_utils.rb:5]: It does not work if I have untracked files

```
$ git status --porcelain
?? .byebug_history
?? .gitignore
?? .idea/
?? .rspec
?? .rubocop.yml
?? .ruby-gemset
?? .ruby-version
```

**Pull request merged**